react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -171,7 +170,7 @@ public boolean onTextContextMenuItem(int id) {
     }
 
     @Override
-    public float getPreformatBackgroundAlpha(@NotNull TypedArray styles) {
+    public float getPreformatBackgroundAlpha(@Nullable TypedArray styles) {
```

Comment:
Looking at the [corresponding Aztec code here](https://github.com/wordpress-mobile/AztecEditor-Android/pull/869/files#diff-7ca71cd1f009ab8438e0901205001b14R715) it looks like the parameter is not nullable (no `?` in the parameter type). Perhaps `@NonNull` is the appropriate annotation here?

Reply:
Hey @hypest, yes you were right in the first place. 
For some reason, I mixed those two annotations.
It's fixed now. Sorry about misunderstanding.